website/docs/execution-node/fee-recipient.md

```diff
@@ -1,83 +1,192 @@
 ---
 id: fee-recipient
-title: Setting an Address for Receiving Transaction Fees
-sidebar_label: Setting an Address for Receiving Transaction Fees
+title: Configuring a Fee Recipient Address 
+sidebar_label: Configuring a Fee Recipient Address 
+doc_type: how-to
+sme: james-prysm
 ---
 
-# Fee Recipient
 
-Validators don't currently receive any portion of the transaction fees paid by end-users when proposing new blocks.
 
-**Fee Recipient** is a feature that addresses this. With Fee Recipient configured, validators will receive a portion of transaction fees when proposing blocks.
+# Configuring a Fee Recipient Address 
 
-This document shows you how to configure a custom address to receive transaction fees using your Prysm validator client software. 
+<!-- alt: ## How to configure Fee Recipient on your client instance and/or beacon node -->
 
-:::tip Post-Merge
-This feature will activate after [The Merge](https://ethereum.org/en/upgrades/merge/).
+**Fee Recipient** is a feature that lets you specify a priority fee recipient address on your validator client instance and beacon node. After [The Merge](https://ethereum.org/en/upgrades/merge/), execution clients will begin depositing priority fees into this address whenever your validator client proposes a new block.
```

Comment:
I wonder if there is a feature to link down to another section so you can explain users what a priortiy fee is. is this something we are using or is this standardized somewhere in existing documentation? 🤔

Reply:
Clarifying -> are you recommending 

1) linking to a glossary definition that defines priority fee? (this is on my radar! but not within the scope of this PR.)
2) inline/in-doc definitions? (this strikes me as an optimization - step 1 define in glossary, step 2 embed from glossary)
3) linking to the explanation of priority fee a couple lines below this copy?

Reply:
well at first I was thinking 1, but realized you described it in the background so i guess 2.